components/Navbar/Navbar.tsx

```diff
@@ -55,6 +55,10 @@ const Navbar = () => {
                         {
```

Comment:
Forgot to show Mentors above when the user is authenticated and registered.

Reply:
Instead, remove adding the Mentors page here and move it to `DEFAULT_NAVBAR_ITEMS` at the top of the screen.